packages/bot-skeleton/jest.config.js

```diff
@@ -0,0 +1,18 @@
+const baseConfigForPackages = require('../../jest.config.base');
+
+module.exports = {
+    ...baseConfigForPackages,
+    clearMocks: true,
+    moduleNameMapper: {
+        '\\.s(c|a)ss$': '<rootDir>/../../__mocks__/styleMock.js',
+        '^.+\\.svg$': '<rootDir>/../../__mocks__/styleMock.js',
+        '^Constants/(.*)$': '<rootDir>/src/constants/$1',
+        '^Scratch/(.*)$': '<rootDir>/src/scratch/$1',
+        '^Services/(.*)$': '<rootDir>/src/services/$1',
+        '^Utils/(.*)$': '<rootDir>/src/utils/$1',
+    },
+    collectCoverageFrom: ['**/*.{js,jsx,ts,tsx}', '!**/node_modules/**'],
```

Comment:
this line is not needed. it's already on `jest.config.base`

Reply:
Good point 👍, it really is, I double-checked and deleted